### ISSUE TITLE
Viewports have white gap between data and border (on hi-dpi screens?)

### ISSUE THREAD

**Issue by fm3** [scalableminds/webknossos#3785]
I would assume the data should go all the way to the CSS-drawn red border, so both the red and the white inside of that are unwanted artifacts.
![viewport-double-frames](https://user-images.githubusercontent.com/1390035/52950081-5830cd80-337e-11e9-812d-099241d79601.png)

**Comment by philippotto** [comment]
Might be high dpi related. I cannot reproduce this on my screen.

**Comment by fm3** [comment]
yeah, looks like it is a hi-dpi issue. it also looks slightly different in firefox and chrome (though neither quite as expected). Let’s keep the issue open in the backlog but I guess the priority is low.

**Comment by philippotto** [comment]
The original issue with the double-border is resolved by now, but there are still white gaps between the data and the outer border :(

![image](https://user-images.githubusercontent.com/2486553/55551588-80c91880-56db-11e9-97a5-1583c674026e.png)

I'll update the issue title.
